### PR TITLE
chore: upgrade requests

### PR DESCRIPTION
**Pull request by pauldambra**
so that we upgrade urllib3 which sentry latest needs

## Problem

running `pip install -r ./requirements.txt` worked fine

but running `pip install -r ./requirements.txt && pip install -r ./requirements-dev.txt` gave an error message

```
Installing collected packages: urllib3
  Attempting uninstall: urllib3
    Found existing installation: urllib3 1.26.13
    Uninstalling urllib3-1.26.13:
      Successfully uninstalled urllib3-1.26.13
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
sentry-sdk 1.11.1 requires urllib3>=1.26.11; python_version >= "3.6", but you have urllib3 1.26.5 which is incompatible.
```

## Changes

Upgraded the requests library and regenerated the requirements files

## How did you test this code?

opening the PR so the tests would run 🤷

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```
Comment (neilkakkar) [inline requirements-dev.txt:2]: git add requirements-dev.in

Comment (pauldambra) [reply requirements-dev.txt:2]: It didn't change... only changes to the generated file because of the `-c requirements.txt`

Comment (neilkakkar) [reply requirements-dev.txt:2]: Gotcha, thanks!

You need to use python 3.8 to generate this though? Don't think we've moved to 3.9 yet.

**Review by neilkakkar — APPROVED** [review]
Not sure about default generated using python 3.9, but otherwise LGTM!

**Comment by pauldambra** [comment]
generating with python 3.8 does add something that isn't there with 3.9 🙉

**Pull request merged**